cmd/meroxa/root/resources/create_test.go

```diff
@@ -145,9 +147,20 @@ Resource %q is successfully created!
 func TestCreateResourceExecutionWithEnvironmentName(t *testing.T) {
 	ctx := context.Background()
 	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
```

Comment:
Out of curiosity, what happens if we don't do this?

Reply:
ah, I was doing this based on a past version of go - <= go.1.14.  these changes are not needed. I'll remove them. 
https://pkg.go.dev/github.com/golang/mock/gomock#Controller.Finish
"Finish checks to see if all the methods that were expected to be called were called. It should be invoked for each Controller. It is not idempotent and therefore can only be invoked once.
New in go1.14+, if you are passing a *testing.T into NewController function you no longer need to call ctrl.Finish() in your test methods."